src/model/Model.cpp

```diff
@@ -6,71 +6,63 @@ Model::Model(string name) {
         delete *it;
     } 
     fields.clear();
 }
 
-IntegerField* Model::integerField(string name, int defaultValue, bool primary, bool unique, bool nullable, bool autoInc)
-{
-    IntegerField * f = new IntegerField(name, defaultValue, primary, unique, nullable, autoInc);
+IntegerField* Model::integerField(string fieldName, int defaultValue, bool primaryKey, bool unique, bool nullable, bool autoIncrement) {
+    IntegerField * f = new IntegerField(fieldName, defaultValue, primaryKey, unique, nullable, autoIncrement);
     fields.push_back(f);
     
     return f;
 }
 
-FloatField* Model::floatField(string name, double defaultValue, bool primary, bool unique, bool nullable)
-{
-    FloatField * f = new FloatField(name, defaultValue, primary, unique, nullable);
+FloatField* Model::floatField(string fieldName, double defaultValue, bool primaryKey, bool unique, bool nullable) {
+    FloatField * f = new FloatField(fieldName, defaultValue, primaryKey, unique, nullable);
     fields.push_back(f);
     
     return f;
 }
 
-TextField* Model::textField(string name, string defaultValue, bool primary, bool unique, bool nullable)
-{
-    TextField * f = new TextField(name, defaultValue, primary, unique, nullable);
+TextField* Model::textField(string fieldName, string defaultValue, bool primaryKey, bool unique, bool nullable) {
+    TextField * f = new TextField(fieldName, defaultValue, primaryKey, unique, nullable);
     fields.push_back(f);
     
     return f;
 }
 
-CharField* Model::charField(string name, int maxLength, string defaultValue, bool primary, bool unique, bool nullable)
-{
-    CharField * f = new CharField(name, maxLength, defaultValue, primary, unique, nullable);
+CharField* Model::charField(string fieldName, int maxLength, string defaultValue, bool primaryKey, bool unique, bool nullable) {
+    CharField * f = new CharField(fieldName, maxLength, defaultValue, primaryKey, unique, nullable);
     fields.push_back(f);
     
     return f;
 }
 
-BooleanField* Model::booleanField(string name, bool defaultValue, bool primary, bool unique, bool nullable)
-{
-    BooleanField * f = new BooleanField(name, defaultValue, primary, unique, nullable);
+BooleanField* Model::booleanField(string fieldName, bool defaultValue, bool primaryKey, bool unique, bool nullable) {
+    BooleanField * f = new BooleanField(fieldName, defaultValue, primaryKey, unique, nullable);
     fields.push_back(f);
     
     return f;
 }
 
-int Model::insert(Mode mode)
-{
+int Model::insert(Mode mode) {
     vector<Model*> models;
     models.push_back(this);
     
     return insertBatch(models, 1, mode);
 }
 
-int Model::insertBatch(vector<Model*> models, int batchsize, Mode mode)
-{
-    int size = models.size();
+int Model::insertBatch(vector<Model*> models, unsigned int batchsize, Mode mode) {
+    unsigned int size = models.size();
     
-    if(size < batchsize)
+    if (size < batchsize)
```

Comment:
Put curly brackets around this if-statement and add that brackets should always be used (even when not necessary) to the README.